src/components/theme/SearchWidget/SearchWidget.jsx

```diff
@@ -95,7 +95,13 @@ class SearchWidget extends Component {
    */
   render() {
     return (
-      <Form action="/search" onSubmit={this.onSubmit}>
+      <Form
+        action={
+          `${this.props?.content?.['@components'].navroot?.navroot?.['@id']}/search` ||
```

Comment:
This is using the ['@components'] expander without knowing if the expander is in place or not.

Reply:
Instead of prop drilling the `content` from App then Header then here, it's not better to call the useSelector for content and get it only here?

Reply:
Agree against prop drilling. With expanders handled correctly, it should be possible to always get this info from the navroot redux store with useSelector